e3sm_diags/run.py

```diff
@@ -233,29 +357,21 @@ def _get_instance_of_param_class(self, cls, parameters):
         msg = "There's weren't any class of types {} in your parameters."
         raise RuntimeError(msg.format(class_types))
 
-    def _get_other_diags(self, run_type):
+    def _get_diags_from_cfg_file(self) -> Union[List, List[CoreParameter]]:
         """
-        If the user has ran the script with a -d, get the diags for that.
-        If not, load the default diags based on sets_to_run and run_type.
+        Get parameters defined by the cfg file passed to -d/--diags (if set).
+
+        If ``-d`` is not passed, return an empty list.
         """
         args = self.parser.view_args()
+        params = []
 
-        # If the user has passed in args with -d.
         if args.other_parameters:
             params = self.parser.get_cfg_parameters(argparse_vals_only=False)
-        else:
-            default_diags_paths = [
-                get_default_diags_path(set_name, run_type, False)
-                for set_name in self.sets_to_run
-            ]
-            params = self.parser.get_cfg_parameters(
-                files_to_open=default_diags_paths, argparse_vals_only=False
-            )
-
-        # For each of the params, add in the default values
-        # using the parameter classes in SET_TO_PARAMETERS.
-        for i in range(len(params)):
-            params[i] = SET_TO_PARAMETERS[params[i].sets[0]]() + params[i]
+            # For each of the params, add in the default values
+            # using the parameter classes in SET_TO_PARAMETERS.
+            for i in range(len(params)):
+                params[i] = SET_TO_PARAMETERS[params[i].sets[0]]() + params[i]
 
         return params
```

Comment:
This function was renamed and I removed the behavior where default diagnostics would run if `-d` is not passed.